java-fundamentals-practical/src/am/javaprogram/fundamentals/greetings/GreetingsApp.java

```diff
@@ -0,0 +1,26 @@
+package am.javaprogram.fundamentals.greetings;
+
+import java.util.Scanner;
+
+public class GreetingsApp {
+     // fields  of  GreetingApp Class
+  private Scanner scanner;
+  private final String EXIT_VALUE = "exit";
+  // printGreeting method  for  printing ""Hello `username`""
+  private void printGreeting() {
+    scanner = new Scanner(System.in);
```

Comment:
Initialize Scanner in the constructor or make it static.
private static final Scanner scanner  = new Scanner(System.in);